docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -735,6 +735,52 @@ class SharedService {
 <2> `@Lock(Lock.Type.READ)` overrides the value specified at class level. It means that any number of clients can invoke the method concurrently, unless the bean instance is locked by `@Lock(Lock.Type.WRITE)`. 
 <3> You can also specify the "wait time". If it's not possible to acquire the lock in the given time a `LockException` is thrown.
 
+=== Repeatable interceptor bindings
+
+Quarkus has limited support for `@Repeatable` interceptor binding annotations.
+
+When binding an interceptor to a component, you can declare multiple `@Repeatable` annotations on methods.
+Repeatable interceptor bindings declared on classes and stereotypes are not supported, because there are some open questions around interactions with the Interceptors specification.
+This might be added in the future.
+
+As an example, suppose we have an interceptor that stores method's return values into a cache.
+The corresponding interceptor binding would be called `@Cached` and would be declared as `@Repeatable`.
+If we wanted to store the return value into two caches at the same time, we would add `@Cached` twice:
+
+[source,java]
+----
+@ApplicationScoped
+class AllEncompassingService {
+  @Cached(cacheName = "foo")
+  @Cached(cacheName = "bar")
+  BigData heavyComputation() {
+    // ...
+  }
```

Comment:
Here's a proposal which is really close from the current Quarkus caching API.

```suggestion
As an example, suppose we have an interceptor that removes all entries from a cache.
The corresponding interceptor binding would be called `@CacheInvalidateAll` and would be declared as `@Repeatable`.
If we wanted to clear two caches at the same time, we would add `@CacheInvalidateAll` twice:

[source,java]
----
@ApplicationScoped
class AllEncompassingService {
  @CacheInvalidateAll(cacheName = "foo")
  @CacheInvalidateAll(cacheName = "bar")
  void clearBothCaches() {
    // ...
  }
```

When this PR is merged, `@CacheInvalidateAll` will even become an interceptor binding (it's a simple annotation right now).

Reply:
OK, I didn't really want to use _existing_ annotations, but this will do, thanks! :-)